test/google_search_data_viewer/keyword/keywords_test.exs

```diff
@@ -50,6 +50,29 @@ defmodule GoogleSearchDataViewer.Keyword.KeywordsTest do
     end
   end
 
+  describe "get_keyword_upload_with_search_results/1" do
+    test "with an existing uploaded keyword and search results, returns the keyword upload with search results for the given id" do
+      keyword_upload = insert(:keyword_upload)
+
+      insert_list(3, :search_result_url, keyword_upload: keyword_upload)
+
+      keyword_upload_result = Keywords.get_keyword_upload_with_search_results(keyword_upload.id)
+
+      assert Enum.count(keyword_upload_result.search_result_urls) == 3
+    end
+
+    test "with an existing uploaded keyword and no search results, returns the keyword upload with search results as an empty list for the given id" do
+      keyword_upload = insert(:keyword_upload)
+      other_keyword_upload = insert(:keyword_upload)
+
+      insert_list(3, :search_result_url, keyword_upload: other_keyword_upload)
```

Comment:
```suggestion
      insert(:search_result_url, keyword_upload: other_keyword_upload)
```

One is enough.

Reply:
Nice spot, inserting 3 here is pointless as I am only checking on the empty list for the other Upload